freqtrade/strategy/interface.py

```diff
@@ -476,8 +481,20 @@ def get_signal(self, pair: str, timeframe: str, dataframe: DataFrame) -> Tuple[b
         (buy, sell) = latest[SignalType.BUY.value] == 1, latest[SignalType.SELL.value] == 1
         logger.debug('trigger: %s (pair=%s) buy=%s sell=%s',
                      latest['date'], pair, str(buy), str(sell))
+        if self.ignore_expired_candle(dataframe=dataframe, buy=buy):
```

Comment:
Consider passing `latest` instead of the full dataframe - it'll contain the last row (the row that's the base for the buy signal) - but it's not calculated via `.tail()` - so allows reordering of the dataframe by the strategy (which i'd advise noone to do - but we're protected from it).

You will also need `timeframe_minutes` in the function (see comment on the function for an explanation why).

Reply:
Correct, I adjusted the code due to these new insights, thanks!